src/GitHub.Exports.Reactive/Services/IPullRequestSessionManager.cs

```diff
@@ -31,6 +31,18 @@ public interface IPullRequestSessionManager : INotifyPropertyChanged
         /// </returns>
         IPullRequestSession CurrentSession { get; }
 
+        /// <summary>
+        /// Ensures that the service is initialized.
+        /// </summary>
+        /// <returns>A task that when completed indicates that the service is initialized.</returns>
+        /// <remarks>
```

Comment:
Simplying? ;-)

Reply:
Hmm, is seems I missed the line with `simplying` and it isn't showing up for me as an in-line comment. What's going on? 😕

Reply:
I don't see the above as inline comments either